src/components/ItemSet/index.js

```diff
@@ -18,22 +19,32 @@ const ITEM_TYPE_COMPONENTS = {
   exoticCatalysts: MasterworkCatalyst
 };
 
-function ItemSet({
-  className,
-  inventory,
-  itemDefs,
-  setPopper,
-  setModal,
-  set,
-  objectiveInstances,
-  objectiveDefs
-}) {
-  const { name, noUi, description, sections, image } = set;
+class ItemSet extends Component{
+  constructor(props) {
+    super(props);
+    this.accordion = this.accordion.bind(this);
+  }
+  
+  accordion(){
+    const key = window.location.pathname + '-' + this.props.set.name;
```

Comment:
I'm changing around set URLs all the time (admittedly, more often than I should), so using `location.pathname` might not be all that reliable. In the commit I linked to, I entertained the idea of a new ID field on each of the sets which would be globally unique, and could be used more reliably...